prow/scripts/cluster-integration/kyma-gke-long-lasting.sh

```diff
@@ -197,11 +196,27 @@ function generateAndExportLetsEncryptCert() {
         --server https://acme-v02.api.letsencrypt.org/directory \
         --dns-google-propagation-seconds=600 \
         -d "*.${DOMAIN}"
-
+	
     TLS_CERT=$(base64 -i ./letsencrypt/live/"${DOMAIN}"/fullchain.pem | tr -d '\n')
     export TLS_CERT
     TLS_KEY=$(base64 -i ./letsencrypt/live/"${DOMAIN}"/privkey.pem   | tr -d '\n')
     export TLS_KEY
+	#encrypt the tls cert
+	gcloud kms encrypt --location global \
+	--keyring kyma-prow \
+	--key projects/kyma-project/locations/global/keyRings/kyma-prow/cryptoKeys/kyma-prow-encryption\
+	--plaintext-file ./letsencrypt/live/"${DOMAIN}"/fullchain.pem  \
+	--ciphertext-file "nightly-aks-tls-integration-app-client-cert.encrypted"
+	#encrypt the tls private key
+	gcloud kms encrypt --location global \
+	--keyring kyma-prow \
```

Comment:
keyring and encryption key name can be injected to the job by preset. See config.yaml how we use it. Update documentation (presets.md).

Reply:
will do